core/src/main/java/org/web3j/protocol/websocket/WebSocketClient.java

```diff
@@ -36,7 +36,7 @@ public void onMessage(String s) {
             log.debug("Received message {} from server {}", s, uri);
             listener.onMessage(s);
         } catch (Exception e) {
-            log.error("Failed to process message '{}' from server {}", s, uri);
+            log.error(String.format("Failed to process message '%s' from server %s", s, uri),e);
```

Comment:
@wenshenjun there is no need to use String.format, using the {} placeholders is equivalent.
Can you please retain those and only add in the exception? Moreover ...
```log.error("Failed to process message '{}' from server {}", s, uri, e);```